src/mscorlib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -19,7 +19,9 @@
     <Compile Include="$(MSBuildThisFileDirectory)Microsoft\Win32\SafeHandles\SafeHandleMinusOneIsInvalid.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Microsoft\Win32\SafeHandles\SafeHandleZeroOrMinusOneIsInvalid.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Action.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\AccessViolationException.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\ApplicationException.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\AggregateException.cs" />
```

Comment:
The implementation of `AggregateException.ToString` in CoreRT looks better: 
- Correct handling of localized strings (CoreCLR version is mixing localizable `>>>` with hardcoded `<<<` 
- Less allocations (CoreCLR version will allocate `O(N^2)` where N is number of inner exceptions)